src/components/Overview.tsx

```diff
@@ -0,0 +1,35 @@
+import { View, StyleSheet } from 'react-native'
+
+import { StyledText } from './StyledText'
+import { RefreshIcon } from './icons/Icons'
+import { colors } from '../config/themes/appThemes'
+
+interface OverviewProps {
+  updateTime: string
+}
+
+export const Overview: React.FC<OverviewProps> = (props) => {
+  const { updateTime } = props
+
+  return (
+    <View>
+      <StyledText style={styles.title} title2 neutral100>
```

Comment:
Con lo anterior te evitas poner estilos en el `StyledText` 
```suggestion
      <StyledText title2 neutral100>
```